src/domain/schema.graphql

```diff
@@ -188,3 +198,13 @@ type ContractInfo @entity {
 	version: String!
 	address: Bytes!
 }
+
+type Event @entity {
```

Comment:
how about adding events : [Event!] @derivedFrom('proposal') to proposal entity ?

Reply:
@jellegerbrandy will it be useful to add that?